docarray/document/mixins/porting.py

```diff
@@ -141,6 +149,8 @@ def to_json(self, protocol: str = 'jsonschema', **kwargs) -> str:
             from google.protobuf.json_format import MessageToJson
 
             return MessageToJson(self.to_protobuf(), **kwargs)
+        elif protocol == 'dynamic':
+            return json.dumps(self.to_dict(protocol=protocol))
```

Comment:
pass the `kwargs` for `json.dumps(.., **kwargs)`

Reply:
dumping is not supported anymore